apps/system/lockscreen/lockscreen.html

```diff
@@ -43,9 +43,15 @@
         <div id="lockscreen-alt-camera" class="lockscreen-icon">
           <div id="lockscreen-alt-camera-button" role="button" data-l10n-id="camera-a11y-button" aria-label="Camera"></div>
         </div>
+        <div id="lockscreen-clock-widget">
         <div id="lockscreen-clock-time"></div>
         <div id="lockscreen-date"></div>
+        </div>
         <div id="lockscreen-charging"></div>
+        <div id="lockscreen-enter-code">
+          <img src="lockscreen/style/images/lockscreen_unlock.png" id="lockscreen-enter-code-icon">
+          <p data-l10n-id="enterpincode">Enter PIN code to unlock </p>
```

Comment:
Just specify the data-l10n-id. Please don't put any default text in the element. (Our build scripts actually pre-process the HTML and pre-localize the files to the default language)